deploy/examples/psp.yaml

```diff
@@ -1,259 +1,3 @@
 ####################################################################################################
```

Comment:
How about just delete this whole file? The PSPs haven't been needed/supported for a long time.